src/components/GlobalHeader/index.js

```diff
@@ -114,6 +114,16 @@ class GlobalHeader extends PureComponent {
                 <Icon type="question-circle-o" />
               </a>
             </Tooltip>
+            <Tooltip title="Feedback">
+              <a
+                target="_blank"
+                className={styles.action}
+                style={{ paddingRight: '30px' }}
```

Comment:
The padding is helping us keep the tooltip pointing towards top. Otherwise the tooltip top rotates to opposite side.

Reply:
Is there a way to add this as a "comment"?